### PR TITLE
Task/dss 13.2.2 publish on pypi

### PR DESCRIPTION
**Pull request by kevanescence**

### CODE REVIEW

**Comment by kevanescence** [comment]
[sc-211867]

**Review by rpainblanc — APPROVED** [review]
- Checked the diff with dip@13.2.2 and found no difference with the PR, all changes there are here too
- Checked pip install
- Checked using the API to query DSS

All good ✅

File: HISTORY.txt [hunk 1]
```diff
@@ -2,6 +2,11 @@ Changelog
 ==========
 
 
+13.2.1 (2024-11-04)
```
Comment (rpainblanc) [inline HISTORY.txt:5]: Typo

```suggestion
13.2.2 (2024-11-04)
```

**Review by rpainblanc — CHANGES_REQUESTED** [review]

**Review by rpainblanc — APPROVED** [review]
LGTM

**Pull request merged**